scripts/check-translation-jsons.ts

```diff
@@ -188,6 +197,48 @@ function validateNoDuplicateKeys({
   }
 }
 
+// get keys while respecting different plural/context key suffixes in different languages.
+function getKeysWithoutSuffix(
+  translations: Record<string, string>,
+  lang: string,
+  file: string
+): string[] {
+  // Map<baseKey, Set<suffixes>>
```

Comment:
Ah, yes, it's a leftover! I had a Map there before and tried to explain the key and value. Will remove.

Reply:
Is this comment leftover or maybe I don't understand it 🤔